ts/Core/Series/Point.ts

```diff
@@ -441,6 +438,19 @@ class Point {
         chart.pointCount--;
     }
 
+    /**
+     * Determine if point is null.
+     *
+     * @private
+     * @function Highcharts.Point#determineIsNull
+     */
+    public determineIsNull(): boolean {
```

Comment:
Do we need a new function? Usually in extensions we just override the `Point.isValid` function.

Reply:
[This logic](https://github.com/highcharts/highcharts/pull/16992/files/e27540ab39c2341724e2e59dfe3c8eeae2a58dfd#diff-185b42d88f432e641196b68d7e7f2da4ac7c3fd3d87ccffa9da96eea6a458941L342-L345) was extracted into determineIsNull function in order to restore points’ graphics after zoom reset. Before the change it would be called only initially resulting in missing points: https://jsfiddle.net/BlackLabel/7nx50sm8/ (click apply zoom then reset zoom)

Reply:
That logic is there because the `isValid` function is not part of the basic Point object, just some extended objects. So instead of wrapping it, I think we should just implement the function on the basic Point.

```ts
public isValid(): boolean {
   return this.x !== null && isNumber(this.y);
}
```